content/applications/inventory_and_mrp/purchase/products/uom.rst

```diff
@@ -70,14 +70,14 @@ and name the category.
    :align: center
    :alt: Create a new units of measure category in Odoo Purchase
 
-The next step is to create the two units of measures. To do so, go to :menuselection:`Configuration
---> Units of Measure`.
+The next step is to create the two units of measures. To do so, click into the Unit of Measure
+category. Under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.
 
 First, create the unit of measure used as the reference point for converting to other units of
-measure inside the category by clicking on *Create*. Name the unit and select the units of measure
-category you just created. For the *Type*, select *Reference Unit of Measure for this category
-type*. Enter the *Rounding Precision* you would like to use. The quantity computed by Odoo is always
-a multiple of this value.
+measure inside the category. Name the unit, and select the units of measure category you just
+created. For the *Type*, select *Reference Unit of Measure for this category type*. Enter the
+*Rounding Precision* you would like to use. The quantity computed by Odoo is always a multiple of
+this value.
```

Comment:
remove the use of "you" and correct the incorrect usage of italics here, please

Reply:
I am not making any other updates to the copy outside of the one implemented suggestion, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.